src/providers/postgres/qgspostgresutils.h

```diff
@@ -127,6 +127,15 @@ class QgsPostgresUtils
     * \since QGIS 3.44
     */
     static bool deleteProjectFromSchema( QgsPostgresConn *conn, const QString &projectName, const QString &schemaName );
+
+    /*
+    * Dump
+    *
+    * \returns true on success
+    *
+    * \since QGIS 3.46
+    */
+    static QString dumpVariantMap( const QVariantMap &variantMap, const QString &title = QString() );
```

Comment:
Should this be renamed to something like `variantMapToHtml`, `dump` sounds like it's for terminal output?

Reply:
Done. I have used function that was previously there as internal one, but the suggested name is better